src/subscriptions/SubscriptionManager.h

```diff
@@ -276,10 +276,10 @@ class SubscriptionManager
     pubTransaction(Backend::TransactionAndMetadata const& blobs, ripple::LedgerInfo const& lgrInfo);
 
     void
-    subAccount(ripple::AccountID const& account, session_ptr& session);
+    subAccount(ripple::AccountID const& account, session_ptr const& session);
```

Comment:
While you are here, maybe rename `session_ptr` to `SessionPtrType` or something like that?